packages/react/src/components/DataTable/TableExpandRow.tsx

```diff
@@ -12,13 +12,26 @@ import { ChevronRight } from '@carbon/icons-react';
 import TableCell from './TableCell';
 import { usePrefix } from '../../internal/usePrefix';
 import { TableRowProps } from './TableRow';
+import deprecate from '../../prop-types/deprecate';
```

Comment:
Looks like we can remove this line, that is not being used 👀 

```suggestion
```

Reply:
@guidari thanks, fixed!